docs/sources/setup-grafana/installation/kubernetes/index.md

```diff
@@ -9,114 +9,645 @@ weight: 500
 
 # Deploy Grafana on Kubernetes
 
-This page explains how to install and run Grafana on Kubernetes (K8S). It uses Kubernetes manifests for the setup. If you prefer Helm, refer to the [Grafana Helm community charts](https://github.com/grafana/helm-charts).
+On this page, you will find instructions for installing and running Grafana on Kubernetes (K8s) using Kubernetes manifests for the setup. If Helm is your preferred option, refer to [Grafana Helm community charts](https://github.com/grafana/helm-charts).
+
+## Before you begin
+
+To follow this guide:
+
+- You need the latest version of [Kubernetes](https://kubernetes.io/) running either locally or remotely on a public or private cloud.
+
+- If you plan to use it in a local environment, you can use various Kubernetes options such as [minikube](https://minikube.sigs.k8s.io/docs/), [kind](https://kind.sigs.k8s.io/), [Docker Desktop](https://docs.docker.com/desktop/kubernetes/), and others.
+
+- If you plan to use Kubernetes in a production setting, it's recommended to utilize managed cloud services like [Google Kubernetes Engine (GKE)](https://cloud.google.com/kubernetes-engine), [Amazon Elastic Kubernetes Service (EKS)](https://aws.amazon.com/eks/), or [Azure Kubernetes Service (AKS)](https://azure.microsoft.com/en-us/products/kubernetes-service/).
+
+## System requirements
+
+This section provides minimum hardware and software requirements.
+
+### Minimum Hardware Requirements
+
+- Disk space: 1 GB
+- Memory: 750 MiB (approx 750 MB)
+- CPU: 250m (approx 2.5 cores)
+
+### Supported databases
+
+For a list of supported databases, refer to [supported databases](/docs/grafana/latest/setup-grafana/installation/#supported-databases).
+
+### Supported web browsers
+
+For a list of support web browsers, refer to [supported web browsers](/docs/grafana/latest/setup-grafana/installation/#supported-web-browsers).
+
+{{% admonition type="note" %}}
+Enable port `3000` in your network environment, as this is the Grafana default port.
```

Comment:
Can you change this port?

Reply:
@usmangt , do you know the answer to this question?

Reply:
So port 3000 is the default on non-containerized env. 

Here it is defined inside the grafana.yaml file. It is up to the user to change it to any other number. Its not hard coded